test/terramate_google.tm.hcl

```diff
@@ -0,0 +1,78 @@
+globals {
+  minimum_provider_version    = "4.0"
+  provider_version_constraint = "~> ${global.minimum_provider_version}"
+
```

Comment:
Let's also right away drop support for provider < 4.0 and terraform < 1.0 in versions.tf